packages/react-storage/src/components/StorageBrowser/context/config.ts

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { ListLocations } from './actions/listLocationsAction';
+import { Permission } from './actions/types';
+
+export interface Config<T = Permission> {
+  listLocations: ListLocations<T>;
+}
+
+export const USE_CONFIG_ERROR_MESSAGE = 'Add me later';
```

Comment:
Maybe put a //@TODO here to find it easier later?

Reply:
Normally yes but need to continue working in this file and will just handle it when working on the store